lib/hca/settings.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HCA
+  module Settings
+    CONFIG = Rails.application.config_for(:health_care_application).freeze
+
+    def self.cert_store(paths)
+      store = OpenSSL::X509::Store.new
+      Array(paths).each do |path|
+        store.add_file(Rails.root.join('config', 'health_care_application', 'certs', path).to_s)
```

Comment:
We add trusted certs to the system-wide trust store so this should go away, although we can phase it out after making sure the devops side of thing is working.